### PR TITLE
fix: dApp staking kaizen

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: added docs link to the 'There are too many contract stake entries for the account' error message
* fix: displaying 'days' or 'day' on the MyStakingCard UI
* feat: added notification mark for claiming rewards

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

* feat: added notification mark for claiming rewards

=Staking=

image

=Project=

image

**Changes**
* fix: added docs link to the 'There are too many contract stake entries for the account' error message

=Before=

image

=After=

image

* fix: displaying 'days' or 'day' on the MyStakingCard UI

(more than 1) days
image

### CODE REVIEW

File: src/components/assets/AnchorLinks.vue [hunk 1]
```diff
@@ -43,7 +63,20 @@ export default defineComponent({
       section?.scrollIntoView({ behavior: 'smooth', block: 'start' });
     };
 
-    return { nativeTokenSymbol, scrollTo };
+    const { rewards } = useDappStaking();
+
+    const isStakerRewards = computed<boolean>(() => {
```
Comment (bobo-k2) [inline src/components/assets/AnchorLinks.vue:68]: Those variables are already defined in `useDappStaking`. Please reuse them
image

File: src/components/assets/YourProject.vue [hunk 2]
```diff
@@ -35,11 +44,40 @@ export default defineComponent({
...
+      if (dapp) {
+        return dapp.rewards > 0;
+      }
+      return false;
+    }
+
+    const setDappRewardsMap = async (): Promise<void> => {
+      const dAppRewardsArray: { dapp: string, rewards: bigint }[] = [];
+      if (props.ownDapps.length === 0) return
+      const stakingV3service = container.get<() => IDappStakingService>(
```
Comment (bobo-k2) [inline src/components/assets/YourProject.vue:64]: You can remove this if you use `getDappRewards`, from `useDappStaking`. My general idea was to move all interaction with services to the hook.

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: src/components/assets/AnchorLinks.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/assets/AnchorLinks.vue:68]: Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1193/commits/6d9e6d81d4cd3119e64d8a2bdec68caa45594cc0) commit.

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Review by gluneau — APPROVED** [review]

**Comment by Kahonnohak** [comment]
As I requested please remove the onboarding modals then let's merge tomorrow

**Pull request merged**